actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserConsentActor.java

```diff
@@ -43,16 +60,22 @@ public void onReceive(Request request) throws Throwable {
     }
 
     private void getUserConsent(Request request) {
+        Map<String, Object> consent = (Map<String, Object>) request.getRequest().getOrDefault(JsonKey.CONSENT, new HashMap<String, Object>());
+        Map<String, Object> filters = (Map<String, Object>) consent.getOrDefault(JsonKey.FILTERS, new HashMap<String, Object>());
+        String userId = (String) filters.getOrDefault(JsonKey.USER_ID, "");
+        String consumerId = (String) filters.getOrDefault(JsonKey.CONSUMERID, "");
+        String objectId = (String) filters.getOrDefault(JsonKey.OBJECTID, "");
+
+        Map<String, Object> getConsentFromDBReq = new HashMap<String, Object>();
+        getConsentFromDBReq.put(JsonKey.CONSENT_USER_ID, userId);
```

Comment:
missing if clause? 
Calling with null or empty consumerId, objectId or userId must be removed. If this is already validated at the controller layer, then this must assert and not have a default.

Reply:
Validated at controller layer. So removing defaults.